pkg/license/license.go

```diff
@@ -187,6 +171,29 @@ func FindLicenseFiles(fsys fs.FS) ([]LicenseFile, error) {
 	return licenseFiles, nil
 }
 
+// IsLicenseFile checks if a file is a license file based on its name.
```

Comment:
I like to leave a brief description of each return value of a function.  For example, it took me a few seconds to find what `weight` represents here.  WDYT?